From b341a1e1fce48012fc5bcf39337488fd33210616 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 3 Jul 2016 12:20:21 +0200
Subject: [PATCH] Remove PIE flags from the build

Generating a statically linked binary built with PIE requires the
Scrt1.o file, which isn't part of Buildroot uClibc toolchains. To
solve this, we simply disable the PIE flags.

[olivier tweak patch for 2.2.4]

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Signed-off-by: Olivier Schonken <olivier.schonken@gmail.com>
[Fabrice: updated for 2.3.0]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 Makedefs.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makedefs.in b/Makedefs.in
index 3afef0a..299b297 100644
--- a/Makedefs.in
+++ b/Makedefs.in
@@ -148,7 +148,7 @@ IPPFIND_BIN	=	@IPPFIND_BIN@
 			$(ONDEMANDFLAGS) $(OPTIONS)
 ALL_DSOFLAGS	=	-L../cups @ARCHFLAGS@ @RELROFLAGS@ $(DSOFLAGS) $(OPTIM)
 ALL_LDFLAGS	=	-L../cups @LDARCHFLAGS@ @RELROFLAGS@ $(LDFLAGS)  \
-			@PIEFLAGS@ $(OPTIM)
+			$(OPTIM)
 ARCHFLAGS	=	@ARCHFLAGS@
 ARFLAGS		=	@ARFLAGS@
 BACKLIBS	=	@BACKLIBS@
-- 
2.7.4

